develop/EPM_Integration_Training/EpmIntegrationTrainingAbout.md

```diff
@@ -0,0 +1,63 @@
+---
+uid: EpmIntegrationTrainingAbout
+---
+
+# About the EPM Integration Training package
+
+## Installation
+
+The EPM Integration Training package provides you with all the necessary components to understand, navigate, and get hands-on with an EPM solution.
+
+If your DataMiner System is connected to dataminer.services, you can deploy the package directly from the [DataMiner Catalog](https://catalog.dataminer.services/details/b661f936-d6e7-447c-baee-f0a5503e75b4). Otherwise, you can download it from the Catalog and execute it on one of the DMAs in your cluster. 
```

Comment:
Downloading is not possible for external users (this feature is only available for Skyline employees). Can we suggest that they contact their Skyline contact to get the package instead?